test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -6,7 +6,12 @@ Import-Module HelpersCommon
 Describe "New-PSSession basic test" -Tag @("CI") {
     It "New-PSSession should not crash powershell" {
         $platformInfo = Get-PlatformInfo
-        if (($platformInfo -eq "alpine") -or ($platformInfo -eq "raspbian")) {
+        if (
+            ($platformInfo.Platform -eq "alpine") -or
+            ($platformInfo.Platform -eq "raspbian") -or
```

Comment:
Can you continue to use `-match "alpine|raspbian"` here?

Reply:
Fixed